openedx_events/enterprise/data.py

```diff
@@ -0,0 +1,54 @@
+"""
+Data attributes for events within the architecture subdomain `enterprise`.
+
+These attributes follow the form of attr objects specified in OEP-49 data
+pattern.
+"""
+import attr
+
+
+@attr.s(frozen=True)
+class LicenseLifecycle:
+    """
+    Event to track changes to subscriptsion.License records.
```

Comment:
1. typo: subscriptsion => subscription
2. Why not "subscription licenses"?
3. Other data objects use a description more like: "Attributes defined for Open edX Subscription License object."